corehq/apps/app_manager/models.py

```diff
@@ -3507,6 +3498,31 @@ def get_filter_value(self, user, ui_filter):
         return None
 
 
+class AncestorLocationTypeFilter(ReportAppFilter):
+    ancestor_location_type_name = StringProperty()
+
+    def get_filter_value(self, user, ui_filter):
+        """
+        A list of locations who have the same ancestor as user param
```

Comment:
I think this should read: get the location that is an ancestor of the user's assigned location with the specified type